src/libs/utils.js

```diff
@@ -13,6 +13,14 @@ export const applyHoverEffects = (e, style) => {
   });
 };
 
+//currently, dars contain a list of datasets (any length) and a list of length 1 of a datasetId
+//go through the list of datasets and get the name of the dataset whose id is in the datasetId list
+export const getNameOfDatasetForThisDAR = (datasets, datasetId) => {
+  let id = datasetId[0].toString();
+  let data = find({"value" : id})(datasets);
```

Comment:
minor nits.
* Both can be `const`s
* Can be simplified a bit. Prefer lodash `head` or `first` over direct array access
  * this also works: `const data = find({"value" : first(datasetId).toString()})(datasets);`
* Could also use a nil check to ensure `datasetId` isn't an empty array. It should never be, otherwise we'd have an invalid DAR, but better to be safe than throw an error.